library/network/src/modules/firewalld_wrapper.rb

```diff
@@ -101,11 +102,84 @@ def remove_port(port_or_range, protocol, interface)
       zone.remove_port(port)
     end
 
+    # Check whether the firewalld service is enable or not
+    #
+    # @return [Boolean] true if it is enable; false otherwise
+    def is_enabled
+      firewalld.enabled?
+    end
+
+    # Return true if the logging config or any of the zones where modified
+    # since read
+    #
+    # @return [Boolean] true if the config was modified; false otherwise
+    def is_modified
+      firewalld.modified?
+    end
+
+    # Evaluate the zone name of an interface
+    #
+    # @param interface [String] interface name
+    #
+    # @return [String] zone name (nil; not found)
+    def zone_name_of_interface(interface)
```

Comment:
np: zone_name_of(interface) is good enough

Reply:
It's fine as it is, but it somehow sounds redundant as the parameter is the interface itself.